### PR TITLE
Update ldd-corral to better handle versioning

### PR DESCRIPTION
**Pull request by jordanpadams**
Code now pulls the `ldd_version_id` and uses that to determine the namespace directory (e.g. `v1` vs. `v2`)

Resolves NASA-PDS/portal-tasks#36

### CODE REVIEW

File: bin/ldds/ldd-corral.py [hunk 1]
```diff
@@ -38,14 +38,11 @@
 # Discipline LDD Dictionary Type Value
 DISC_LDD_DICT_TYPE = 'Discipline'
 
-STAGING_PATH = '/data/tmp/ldd-staging'
-OUTPUT_PATH = '/data/tmp/ldd-release'
+STAGING_PATH = '/tmp/data/tmp/ldd-staging'
+OUTPUT_PATH = '/tmp/data/tmp/ldd-release'
```
[the remark this answers is not in the document]
Comment (jordanpadams) [reply bin/ldds/ldd-corral.py]: Fixed

File: bin/ldds/ldd-corral.py [hunk 2]
```diff
@@ -38,14 +38,11 @@
 # Discipline LDD Dictionary Type Value
 DISC_LDD_DICT_TYPE = 'Discipline'
 
-STAGING_PATH = '/data/tmp/ldd-staging'
-OUTPUT_PATH = '/data/tmp/ldd-release'
+STAGING_PATH = '/tmp/data/tmp/ldd-staging'
```
[the remark this answers is not in the document]
Comment (jordanpadams) [reply bin/ldds/ldd-corral.py]: Fixed

**Review by c-suh — APPROVED** [review]
@jordanpadams yay thank you! The changes look good! Just in case, I pulled and tested the changes on gamma, and all works as fully intended. Would you like the other reviewers to approve before merging?

**Comment by jordanpadams** [comment]
@c-suh nope. I need to remove that from our config, so your review is sufficient

**Pull request merged**